CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 ### Changed
+- Improve how includes with relative paths are resolved during parsing/compilation.
```

Comment:
I think it should be clearer on how they are improved. Also, this change should be documented - for example it is not clear which path takes priority if there is a file available relative to the current file and another one relative to cwd